src/Umbraco.Web.BackOffice/Controllers/CurrentUserController.cs

```diff
@@ -166,6 +163,10 @@ public IEnumerable<UserTourStatus> GetUserTours()
             var userTours = JsonConvert.DeserializeObject<IEnumerable<UserTourStatus>>(_backofficeSecurityAccessor.BackOfficeSecurity.CurrentUser.TourData);
             return userTours;
         }
+        public IEnumerable<UserData> GetUserData()
+        {
+            return _userDataService.GetUserData();
+        }
```

Comment:
```suggestion
            var userTours = JsonConvert.DeserializeObject<IEnumerable<UserTourStatus>>(_backofficeSecurityAccessor.BackOfficeSecurity.CurrentUser.TourData);
            return userTours;
        }

        public IEnumerable<UserData> GetUserData()
        {
            return _userDataService.GetUserData();
        }
```

Whitespace is your friend 😄